osidb/migrations/0186_map_user_id_to_user_email_or_name.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations
+from django.contrib.auth.models import User
+
+
+def forwards_func(apps, schema_editor):
+
+    context_model = next(
+        (
+            model
+            for model in apps.get_models("osidb")
+            if model._meta.model_name == "context"
+        ),
+        None,
+    )
```

Comment:
I am maybe missing something but this construction to get the model class reference seems too complicated to me. I am looking at some previous migrations and getting the reference seems more straightforward eg.

https://github.com/RedHatProductSecurity/osidb/blob/master/osidb/migrations/0082_migrate_flawacknowledgment_flawmeta.py#L27

Or is there anything special about the **context** model class that it needs a different approach?

Reply:
When I tried `context_model = apps.get_model("osidb", "Context")` where `Context` is the model name, I was getting errors about the model not being registered on django app `"osidb"`, despite the fact that `.get_models` returns it. The other models returned from `.get_models` have `__name__` fields that work with with `.get_model("osidb", <<model_name>>)` but pghistory's context did not, I would guess because it's associated to the app indirectly via the pghistory tracking model decorators